templates/vue/src/utils/Helpers.js

```diff
@@ -143,8 +143,13 @@ export default class Helpers {
       return true
     }
 
-    // Check 2: User is the author of the node
-    if (node.author && wpData.currentUser.ID == parseInt(node.author.id)) {
+    // Check 2: User is the author of the node, if the node is not an accepted or submitted node
+    if (
+      node.author &&
+      wpData.currentUser.ID == parseInt(node.author.id) &&
+      node.status !== "accept" &&
+      node.status !== "submitted"
```

Comment:
These conditions are very confusing. Why are accepted and submitted nodes not permitted, but the other ones are? Can you explain that in a comment to clarify or otherwise think how you can put this code in a way that is easier to understand?

Reply:
Clarified with minor change